oaipmh/admin.py

```diff
@@ -76,3 +76,38 @@ def get_identifier(self, obj):
         return obj.record.identifier
     get_identifier.short_description = 'identifier'
     get_identifier.admin_order_field = 'record__identifier'
+
+
+class MatterhornRecordInline(admin.TabularInline):
+    model = models.MatterhornRecord
+    can_delete = False
+    verbose_name_plural = "Records"
+
+
+@admin.register(models.Series)
+class SeriesAdmin(admin.ModelAdmin):
```

Comment:
is it useful to have the reverse from Tracks? and Playlist/

Reply:
Tracks yes. Playlist, I'll have to see if there's a clean way.

Reply:
I've not done this yet as I've got no way to test it.